tmt/steps/report/html.py

```diff
@@ -121,6 +120,18 @@
 </html>
 """.strip()
 
+def import_jinja2():
+    """
+    Import jinja2 module only when needed
+
+    Until we have a separate package for each plugin.
+    """
+    global jinja2
+    try:
+        import jinja2
+    except ImportError:
+        raise tmt.utils.ReportError(
+            "Install 'jinja2' to use this report method.")
```

Comment:
Or shall we just add `jinja2` to the `install_requires` in `setup.py`? Seems it does not bring make deps.

Reply:
I like an idea to have "base" tmt very minimal. Report.html is its own rpm now so why to require unnecessary deps.
On second thought message could be different e.g.  a pip install hint for subpackage?  (pip install tmt[report-html])

Reply:
Understood, makes sense. Let's keep it small. And agree with the `pip install ...` hint.

Reply:
I'll rebase on top of the current #462 and adjust the message in next couple of minutes.